src/ppx/grfx/vk/vk_device.cpp

```diff
@@ -987,6 +1013,12 @@ bool Device::FragmentStoresAndAtomicsSupported() const
     return mDeviceFeatures.fragmentStoresAndAtomics == VK_TRUE;
 }
 
+bool Device::PartialDescriptorBindingsSupported() const
+{
+    return mDescriptorIndexingFeatures.descriptorBindingPartiallyBound &&
+           mDescriptorIndexingFeatures.runtimeDescriptorArray;
```

Comment:
I am curious, is runtimeDescriptorArray feature required for partial descriptor bindings?

Reply:
Actually no - will remove this.